packages/p2p/src/components/orders/chat/chat-message-receipt.jsx

```diff
@@ -0,0 +1,38 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { Icon } from '@deriv/components';
+import ChatMessage from 'Utils/chat-message';
+
+const ChatMessageReceipt = ({ chat_channel, message, sendbird_user_id }) => {
+    let icon_name;
+
+    if (message.status === ChatMessage.STATUS_PENDING) {
+        icon_name = 'IcMessagePending';
+    } else if (message.status === ChatMessage.STATUS_ERRORED) {
+        icon_name = 'IcMessageErrored';
+    } else {
+        const channel_user_ids = Object.keys(chat_channel.cachedReadReceiptStatus);
+        const other_sendbird_user_id = channel_user_ids.find(user_id => user_id !== sendbird_user_id);
```

Comment:
I think you mean the other user's sendbird id? If so, this wasn't so clear.

```suggestion
        const other_user_sendbird_id = channel_user_ids.find(user_id => user_id !== sendbird_user_id);
```

Reply:
Sure, makes sense. 👍

Reply:
I don't really agree, the `id` is referring to a `sendbird_user` not a `sendbird`. 🤔

Reply:
Ahh I see, then how about `sendbird_other_user_id`, similar to `sendbird_user_id`.